package.json

```diff
@@ -75,38 +73,43 @@
   },
   "dependencies": {
     "attr-accept": "^1.1.3",
-    "prop-types": "^15.5.7"
+    "prop-types": "^15.6.2"
   },
   "devDependencies": {
-    "@commitlint/cli": "^3.2.0",
-    "@commitlint/config-angular": "^3.0.3",
-    "@commitlint/prompt": "^3.2.0",
-    "@commitlint/prompt-cli": "^3.2.0",
+    "@commitlint/cli": "^7.0.0",
+    "@commitlint/config-angular": "^7.0.1",
+    "@commitlint/prompt": "^7.0.0",
+    "@commitlint/prompt-cli": "^7.0.0",
     "babel-cli": "^6.9.0",
-    "babel-core": "^6.9.1",
-    "babel-eslint": "^7.1.1",
-    "babel-jest": "^21.0.0",
+    "babel-core": "^6.26.3",
+    "babel-eslint": "*",
+    "babel-jest": "^23.4.2",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-external-helpers": "^6.22.0",
-    "babel-preset-env": "^1.6.0",
+    "babel-preset-env": "^1.7.0",
     "babel-preset-react": "^6.3.13",
     "babel-preset-stage-1": "^6.24.1",
     "babel-register": "^6.9.0",
-    "commitizen": "^2.9.6",
-    "cross-env": "^5.1.3",
+    "commitizen": "^2.10.1",
+    "cross-env": "^5.2.0",
     "enzyme": "^2.6.0",
-    "eslint": "^4.6.1",
-    "eslint-config-okonet": "^5.0.1",
-    "eslint-config-prettier": "^2.4.0",
-    "eslint-plugin-prettier": "^2.2.0",
+    "eslint": "4.x",
+    "eslint-config-okonet": "^7.0.2",
+    "eslint-config-prettier": "2.x",
+    "eslint-plugin-flowtype": "2.x",
+    "eslint-plugin-import": "2.x",
+    "eslint-plugin-jsx-a11y": "5.x",
+    "eslint-plugin-node": "5.x",
+    "eslint-plugin-prettier": "2.x",
+    "eslint-plugin-react": "7.x",
     "husky": "^0.14.3",
     "imagemin-cli": "^3.0.0",
     "imagemin-pngquant": "^5.0.0",
-    "jest": "^21.0.1",
+    "jest": "^23.5.0",
     "jest-enzyme": "^3.8.2",
-    "lint-staged": "^4.1.0",
-    "markdownlint-cli": "^0.3.1",
-    "prettier": "^1.6.1",
+    "lint-staged": "^7.2.2",
+    "markdownlint-cli": "^0.13.0",
+    "prettier": "*",
```

Comment:
Same here. It's better to know which version you use. `yarn upgrade-interactive --latest` will let you upgrade easier.

Reply:
Same here